### DISCUSSION TITLE
Will there be easier way to manage licenses/users using keygen dashboard?

### DISCUSSION THREAD

**Discussion by JiaPai12138** [keygen-sh/community#32] [Q&A] [answered]
Could there be a way to manage multiple licenses at the same time (like using checkbox), or create multiple licenses following to certain product & policy at one time (instead of manually create it one by one)? Or are those available in paid tiers?

**Comment by ezekg** [comment, marked as answer]
Not available at the moment. In the upcoming dashboard redesign, I would like to support bulk operations like this. But right now, it's not on the immediate roadmap so no timeline yet (or promises) on that.
